appveyor.yml

```diff
@@ -6,12 +6,22 @@ before_build:
   - nuget restore nunit.sln
 
 build_script: 
-  - msbuild nunit.sln /p:Configuration=Release /t:Rebuild
+  - ps: |
+      .\appveyor.ps1
```

Comment:
Correct. Most of my builds call a `build.cmd` or `build.ps1` or `build.sh` that simply delegates to FAKE.exe. FAKE has support for AppVeyor. I helped put it there. Here is a C# project using it:

https://github.com/octokit/octokit.net/blob/master/appveyor.yml
https://github.com/octokit/octokit.net/blob/master/build.fsx